### ISSUE TITLE
Blog references are not correctly handled in search 

### ISSUE THREAD

**Issue by Didier-Lalli** [hpe-dev-incubator/hpe-dev-portal#2809]
When searching for a topic for example "webhook" you find 1 result which is Lionel's blog on COM Webhooks, but when you select it you see an empty blog (just the meta data)

![image](https://github.com/user-attachments/assets/ed5c2f77-5428-4a46-83d3-03f706de3992)

Instead of the full external blog on: https://jullienl.github.io/Implementing-webhooks-with-COM/

We need to apply the same logic that is applied in the blog page when clicking on one of those blog reference.

**Comment by choukden** [comment]
This is true for all the blog link cards (Chapel, Determined AI, etc.) in the "Search" feature.

**Comment by chethan-b-hpe** [comment]
@Didier-Lalli @choukden , I had raised PR for this Issue : https://github.com/hpe-dev-incubator/hpe-dev-portal/pull/2869 , please validate once : https://deploy-preview-2869--hpe-dev-portal.netlify.app/
